src/index.tsx

```diff
@@ -1,13 +1,49 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './pages/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { PeopleProvider } from './contsxts/PeopleContext';
+import { ErrorProvider } from './contsxts/ErrorContext';
+import { LoaderProvider } from './contsxts/LoaderContext';
+import { PeopleFilteredProvider } from './contsxts/PeopleFilteredContext';
```

Comment:
The import path for `PeopleFilteredContext` is incorrect. It should be './contexts/PeopleFilteredContext' instead of './contsxts/PeopleFilteredContext'.